forgerock-ui-user/pom.xml

```diff
@@ -91,13 +97,13 @@
                     </goals>
                     <configuration>
                         <artifactItems>
-                        <artifactItem>
-                            <groupId>org.forgerock.commons.ui</groupId>
-                            <artifactId>forgerock-ui-commons</artifactId>
-                            <version>${project.version}</version>
-                            <type>zip</type>
-                            <classifier>www</classifier>
-                        </artifactItem>
+                            <artifactItem>
+                                <groupId>org.forgerock.commons.ui</groupId>
+                                <artifactId>forgerock-ui-commons</artifactId>
+                                <version>${project.version}</version>
+                                <type>zip</type>
+                                <classifier>www</classifier>
+                            </artifactItem>
                         </artifactItems>
                     </configuration>
                     </execution>
```

Comment:
Seems to be a minor indentation issue.

Reply:
@siepkes take a second look. FR's original indentation was wrong.

Reply:
Ah right, then were good to go!